packages/components/time-picker/src/utils.ts

```diff
@@ -1,7 +1,14 @@
 import dayjs from 'dayjs'
+import customParseFormat from 'dayjs/plugin/customParseFormat'
+import weekOfYear from 'dayjs/plugin/weekOfYear'
+
 import { isArray, isDate, isEmpty } from '@element-plus/utils'
 
 import type { Dayjs } from 'dayjs'
+
+dayjs.extend(weekOfYear)
+dayjs.extend(customParseFormat)
```

Comment:
Maybe we should not add additional plugins into the code, but let users import them when needed.